examples/eks-cluster-with-new-vpc/README.md

```diff
@@ -20,7 +20,7 @@ Ensure that you have installed the following tools in your Mac or Windows Laptop
 
 > **Note**: The policy resource is set as `*` to allow all resources, this is not a recommended practice.
 
-You can find the policy [here](min-iam-policy.json)
+You can find the policy [here](../../docs/iam/min-iam-policy.json)
```

Comment:
this will lead to the "main" min iam policy, which is mix of all the examples (alot of permissions that are not required for this example).
if you compare between the min iam policies of the files, they shouldn't be the same therefore not duplicates, they are result of iamlive output per example

Reply:
np! that's a good feedback for us to better explain the policy for each example in the READMEs.
LGTM